pkg/config/constants/constants.go

```diff
@@ -211,5 +211,5 @@ const (
 	// NoTraffic indicates that no traffic should go through the intended waypoint.
 	NoTraffic = "none"
 
-	EnableV2AutoAllocationLabel = "networking.istio.io/enable-autoallocate-ip"
+	DisableV2AutoAllocationLabel = "networking.istio.io/disable-autoallocate-ip"
```

Comment:
nit: I would keep this as enabled, and have users do `enable: false`.

That way when we have the global controls, we don't have to have users do `disable: false` which is a weird double negative

Reply:
makes sense